sdks/java/io/elasticsearch-tests/elasticsearch-tests-2/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -63,7 +63,7 @@
   @BeforeClass
   public static void beforeClass() throws IOException {
     ServerSocket serverSocket = new ServerSocket(0);
-    int esHttpPort = serverSocket.getLocalPort();
```

Comment:
same, not needed if you apply comments below

Reply:
yes, it can be done like the other tests even with the index mapping overloading => I'll do it myself on another PR